pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -296,17 +297,21 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateClusterConfig(cephFilesyst
 		},
 	}
 
+	csiHostNetworkEnabled, err := strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", "false"))
+	if err != nil {
+		return errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_HOST_NETWORK'")
+	}
```

Comment:
i think you can use `csi.CSIParam.EnableCSIHostNetwork` here instead of rereading the value.